google/resource_compute_instance.go

```diff
@@ -551,6 +553,9 @@ func resourceComputeInstance() *schema.Resource {
 				Deprecated: "Use timeouts block instead.",
 			},
 		},
+		CustomizeDiff: customdiff.All(
+			suppressEmptyGuestAcceleratorDiff,
```

Comment:
Use https://godoc.org/github.com/hashicorp/terraform/helper/customdiff#IfValueChange here so we can chain other customize diff in the future.